lib/LineFollower/MotorSpeedCalibrationState.cpp

```diff
@@ -249,11 +250,14 @@ void MotorSpeedCalibrationState::phase4Finished(StateMachine& sm)
     }
     else
     {
-        Serial.print("Calibrated max. speed: ");
-        Serial.print(maxSpeed);
-        Serial.print(" steps/s (= ");
-        Serial.print(maxSpeed / RobotConstants::ENCODER_STEPS_PER_MM);
-        Serial.println(" mm/s)");
+        char msg[Logging::MESSAGE_BUFFER_SIZE] = "";

```

Comment:
buffer with snprintf not necessary, because the Logging supports variable arguments, which copies again everything to a buffer.